api/policyreport/v1alpha2/common.go

```diff
@@ -101,6 +101,10 @@ type PolicyReportResult struct {
 	// +optional
 	Rule string `json:"rule,omitempty"`
 
+	// PolicyException is the name of the policy exception if exists
+	// +optional
+	PolicyException string `json:"policyException,omitempty"`
```

Comment:
Do we need to coordinate this change with the wg-policy to add this new field?

Reply:
I have no idea. 
cc @JimBugwadia

Reply:
@MariamFahmy98 - the PolicyReport CRD is published by the Policy WG, so we cannot change it directly.

I suggest adding the exception name in the `properties` map instead:

https://github.com/kyverno/kyverno/blob/36c24196fdb32cfeb8073ca11f3102232cd4ec04/api/policyreport/v1alpha2/common.go#L129

Reply:
Also, don't we need the name and the namespace for the exception?

Reply:
The goal behind adding the exception in the reports API is to make it possible to re-apply the policy to the resource considering the newly created exception. Because reports controller doesn't do that unless there are changes in the policy or resource. And it compares those changes against the existing report. Hence, adding an exception won't result of re-applying the policy as the reports controller will compare the policy/resource with the existing report and it will find no changes, then will skip the re-applying process without taking exception into consideration. That's why I thought to add exceptions in the reports API. I amn't sure if we need to add the exception namespace though.

Reply:
IMO since the exceptions in the reports API isn't used to retrieve them during the re-applying process, adding exception namespace won't be necessary unless there is a use case I amn't aware of.

Reports controller listens on exceptions, and whenever there is exception created/updated/deleted, the `reconcile` will be triggered and we list all existing exceptions anyways.